lib/lda.js

```diff
@@ -272,6 +274,17 @@ var lda = new function() {
         }
         return phi;
     }
+
+    this.getRandom = function() {
+        // generage a pseudo-random number using a seed to ensure reproducable results.
+        if (this.RANDOM_SEED) {
+            var x = Math.sin(this.RANDOM_SEED++) * 10000;
```

Comment:
Suggest changing the multiplier to a larger value, 1000000. This gives a more uniform distribution of random values. (10000 is too small, and results in more values around 0 and 1) :)